src/core/src/enclave_sgx.rs

```diff
@@ -187,15 +188,17 @@ impl EnclaveConfig {
         -> bool
     {
         let mut eid:r_sgx_enclave_id_t =  eid;
-        let ret = ffi::verify_wait_certificate(&mut eid, &wait_cert.as_str(), &wait_cert_sign.as_str(), &poet_pub_key.as_str());
-        println!("status {:?}", ret);
+        let ret = ffi::verify_wait_certificate(&mut eid, &wait_cert.as_str(),
+                            &wait_cert_sign.as_str(), &poet_pub_key.as_str());
```

Comment:
This could be arranged like this:
ffi::verify_wait_certificate(&mut eid, 
                                        &wait_cert.as_str(),
                                        &wait_cert_sign.as_str(),
                                        &poet_pub_key.as_str());